tempesta_fw/http_parser.c

```diff
@@ -5107,58 +4641,54 @@ tfw_http_parse_resp(void *resp_data, unsigned char *data, size_t len,
 	}
 
 	/* 'Age:*OWS' is read, process field-value. */
-	TFW_HTTP_PARSE_RAWHDR_VAL(Resp_HdrAgeV, Resp_I_Age, resp,
-				  __resp_parse_age);
+	TFW_HTTP_PARSE_RAWHDR_VAL(Resp_HdrAgeV, resp, __resp_parse_age);
 
 	/* 'Cache-Control:*OWS' is read, process field-value. */
-	TFW_HTTP_PARSE_RAWHDR_VAL(Resp_HdrCache_ControlV, Resp_I_CC, resp,
+	TFW_HTTP_PARSE_RAWHDR_VAL(Resp_HdrCache_CtrlV, resp,
```

Comment:
All other state names look like the header names themselves. `Resp_HdrCache_CtrlV` is now different.

Reply:
Yes, but it's to long and we don't use the state names for atimatic code generation.

Reply:
ok